docs/core/deploying/index.md

```diff
@@ -219,58 +218,192 @@ A complete sample `project.json` file appears later in this section.
+
+```json
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0",
+    "Newtonsoft.Json": "8.0.3"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": "dnxcore50"
+    }
+  },
+  "runtimes": {
+    "win10-x64": {},
+    "osx.10.10-x64": {}
+  }
+}
+```
 
 When you deploy your application, any third-party dependencies used in your app are also contained with your application files. Third-party libraries do not already have to be present on the system on which the app is running.
 
 Note that you can only deploy a self-contained app with a third-party library to platforms supported by that library.
 
+### Deploying a Self-Contained App with a Smaller Footprint ###
+
+If the availability of adequate storage space on target systems is likely to be an issue, you can reduce the overall footprint of your app by excluding some system components. To do this, you explicitly define the .NET Core components that your app includes in your project.json file.
+
+To create a self-contained app with a smaller footprint, start by following the first two steps for creating a self-contained app. Once you've run the `dotnet new` command and added the C# source code to your app, do the following:
+
+1. Open the `project.json` file and replace the `frameworks` section with the following:
+
+    ```json
+    "frameworks": {
+      "netstandard1.6": { }
+    }
+    ```
+This does two things:
+
+    * It indicates that, instead of using the entire `netcoreapp1.0` framework, which includes .NET Core CLR, the .NET Core Library, and a number of other system components, our app uses only the .NET Standard Library.
+
+    * By removing the `"type": "platform"` attribute, it indicates that the framework is provided as a set of components local to our app, rather than as a system-wide platform package.
+
+2. Replace the `dependencies` section with the following:
+
+    ```json
+    "dependencies": {
+      "NETStandard.Library": "1.6.0",
+      "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
+      "Microsoft.NETCore.DotNetHostPolicy":  "1.0.1"
+    },
+    ```
+   This defines the system components used by our app. The system components packaged with our app include the .NET Standard Library, the .NET Core runtime, and the .NET Core host. This produces a self-contained app with a smaller footprint than if you had simply modified the `frameworks` section to indicate that your app is providing the entire `netcoreapp1.0` framework.
+
+3. Create a `runtimes` section in your `project.json` file that defines the platforms your app targets and specify the runtime identifier of each platform that you target. See [Runtime IDentifier catalog](../rid-catalog.md) for a list of runtime identifiers. For example, the following `runtimes` section indicates that the app runs on 64-bit Windows 10 operating systems and the 64-bit OS X Version 10.10 operating system.
```

Comment:
Nit: maybe add something like "as in the previous case" just to make sure people understand that the `runtimes` node does the same thing.